packages/admin/admin/src/index.ts

```diff
@@ -78,6 +78,7 @@ export { ErrorDialog, ErrorDialogOptions, ErrorDialogProps } from "./error/error
 export { ErrorDialogHandler } from "./error/errordialog/ErrorDialogHandler";
 export { ErrorScope, errorScopeForOperationContext, LocalErrorScopeApolloContext } from "./error/errordialog/ErrorScope";
 export { useErrorDialog, UseErrorDialogReturn } from "./error/errordialog/useErrorDialog";
+export { InlineError } from "./error/inlineError/InlineError";
```

Comment:
Please also export the Props and ClassKey types

Reply:
added it.